### PR TITLE
added lightupChinatown donation progress bar UI

### PR DESCRIPTION
**Pull request by FelicityZhang**
I added the UI for the progress bar. We still need to connect to a backend route that serves up the amount raised, but I think once that is in there this should work.

### CODE REVIEW

**Comment by danthaman44** [comment]
The code LGTM, can you attach some screenshots of what it looks like on desktop and mobile?

**Comment by danthaman44** [comment]
Great work on this!

File: src/components/LightUpChinatown/LightUpChinatownPage.tsx [hunk 1]
```diff
@@ -14,6 +15,16 @@ const LightUpChinatownPage = () => {
...
   const timeUntilEnd = campaignEndDate.getTime() - today.getTime();
   const daysUntilEnd = Math.ceil(timeUntilEnd / (1000 * 3600 * 24));
+  const [contributions, setContributions] = useState(0);
+
+  useEffect(() => {
+    setContributions(15000);
+    // TODO: use the below after backend route to fetch raised is implemented
+    //
+    // const fetchData = async () => {
+    // setContributions(data.contributions.raised);
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/LightUpChinatownPage.tsx:25]: PR for this here! https://github.com/sendchinatownlove/ruby/pull/283

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -209,6 +209,9 @@
     "campaignHeader": "Light Up Chinatown Fundraising Campaign:",
     "campaignDates": "November 1 - 30, 2020",
     "campaignDaysLeft": "More Days Left",
+    "progressBarHeader": "Total Raised: ",
+    "raiseGoal1": " GOAL 1: $23.5k",
+    "raiseGoal2": " GOAL 2: $47k",
```
Comment (justintmckibben) [inline src/locales/en/translation.json:214]: mind adding into the chinese version of the file too? (no need to translate it, just the english is fine)

File: src/components/LightUpChinatown/LightUpChinatownPage.tsx [hunk 3]
```diff
@@ -214,6 +228,14 @@ const CampaignInfoTime = styled(CampaignInfoText)`
   font-size: 18px;
 `;
 
+const DonationProgress = styled.div`
+  position: relative;
+  // background: red;
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/LightUpChinatownPage.tsx:233]: forgot to delete this

File: src/components/LightUpChinatown/DonationProgressBar.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+const GoalImage1 = styled.div`
+  width: 25px;
+  height: 25px;
+  background: #ffffff;
+  border: 4px solid #ff3a2f;
+  box-sizing: border-box;
+  position: absolute;
+  left: 50%;
+  right: 50%;
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/DonationProgressBar.tsx:72]: Could you use flexbox instead to center?

File: src/components/LightUpChinatown/DonationProgressBar.tsx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  border-radius: 12px;
+`;
+const GoalImage2 = styled(GoalImage1)`
+  left: 98%;
```
Comment (justintmckibben) [inline src/components/LightUpChinatown/DonationProgressBar.tsx:76]: Instead of using percent, mind adding padding/margins in pixels?

Comment (FelicityZhang) [reply src/components/LightUpChinatown/DonationProgressBar.tsx:76]: If I add padding/margins, the progress bar will not be responsive. 
Cause we will have Filler, so I want to make the image have the absolute position.

**Comment by FelicityZhang** [comment]
> The code LGTM, can you attach some screenshots of what it looks like on desktop and mobile?
sure!
mobile version
progress bar

File: src/components/LightUpChinatown/DonationProgressBar.tsx (hunk 5 above)
Comment (justintmckibben) [reply src/components/LightUpChinatown/DonationProgressBar.tsx:75]: There are ways to making this responsive with the use of padding/margin in combination with fill/width: 100% without needing absolute

Comment (FelicityZhang) [reply src/components/LightUpChinatown/DonationProgressBar.tsx:75]: I see, I will try to figure it out. Thanks! : )

**Pull request merged**